programs/svm-spoke/src/event.rs

```diff
@@ -0,0 +1,107 @@
+use anchor_lang::prelude::*;
+
+// Admin events
+#[event]
+pub struct SetXDomainAdmin {
+    pub new_admin: Pubkey,
+}
+
+#[event]
+pub struct PausedDeposits {
+    pub is_paused: bool,
+}
+
+#[event]
+pub struct PausedFills {
+    pub is_paused: bool,
+}
+
+#[event]
+pub struct EnabledDepositRoute {
+    pub origin_token: Pubkey,
+    pub destination_chain_id: u64,
+    pub enabled: bool,
+}
+
+// Deposit events
+#[event]
+pub struct V3FundsDeposited {
+    pub input_token: Pubkey,
+    pub output_token: Pubkey,
+    pub input_amount: u64,
+    pub output_amount: u64,
+    pub destination_chain_id: u64,
+    pub deposit_id: u32,
```

Comment:
I think we also might need to change `number_of_deposits` counter in state to u32